src/editor.js

```diff
@@ -298,6 +298,17 @@ const PreviewPanel = ({
   </div>
 `;
 
+const dispatchRefreshIcon = redispatchAs(g('refresh-icon'));
+
+const RefreshIcon = () => svg`
+  <svg @click=${dispatchRefreshIcon} xmlns="http://www.w3.org/2000/svg" cursor="pointer" width="24px" height="24px" viewBox="0 0 24 24" fill="#000000">
```

Comment:
Let's wrap the `svg` in a `div` with the dispatcher, so we can style it with `cursor: pointer`.

And let's rename the dispatcher to `dispatchUpdatePreview`.

```js
const RefreshIcon = () => html`
  <div @click=${dispatchUpdatePreview}>
    ${svg`
      <svg ...>
    `}
  </div>
`;
```